pages.fa/linux/apk.md

```diff
@@ -0,0 +1,28 @@
+# apk
+
+> ابزار مدیریت بسته آلپاین لینوکس.
+> اطلاعات بیشتر: <https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management>.
+
+- آپدیت فهرست مخزن ها از تمام مخازن ریموت:
+
+`apk update`
+
+- نصب یک بسته جدید:
+
+`apk add {{package}}`
+
+- حذف یک بسته:
+
+`apk del {{package}}`
+
+- تعمیر یک بسته یا ارتقا آن بدون تغییر دادن وابستگی های اصلی:
+
+`apk fix {{package}}`
+
+- جستجوی یک بسته با کلمات کلیدی:
+
+`apk search {{keywords}}`
+
+- نمایش اطلاعات درمورد بسته مورد نظر:
+
+`apk info {{package}}`
```

Comment:
```suggestion
`apk info {{بسته}}`
```

Reply:
Yes sure, i didn't translate arguments to Farsi because i noticed when we have two (or more) arguments, the order is disarranged, and this is because Farsi is an RTL language, are you have any suggestion for this issue?

example:
mv {[old_name}] {{new_name}}
mv نام_قدیمی نام_جدید

which arguments is totally reversed when printing in terminal.

Reply:
Oh, this is the first time I have heard about the issue. I can indeed confirm this issue on our web client too https://tldr.inbrowser.app/pages.fa/linux/adduser, whereas the page is fine on GitHub. What do you think, should we use English placeholders across Farsi pages? (If so I will fix them, and note the same in common-arguments file too)

Reply:
Seems like Arabic pages are affected too (i.e. https://tldr.inbrowser.app/pages.ar/linux/gpasswd), IG for both languages we can use English placeholders [will fix it now].

Reply:
The problem of LTR and RTL is an constant problem we have such problems even in big softwares like Word.
also see https://tldr.inbrowser.app/pages.fa/common/dd, in the examples you see a word like IMG is in the middle of text and the rest of text placed in the right hand of IMG !
![image](https://github.com/tldr-pages/tldr/assets/87926618/dd722692-b899-46fe-a0ce-01087020711b)
Correct form:
![image](https://github.com/tldr-pages/tldr/assets/87926618/3f57a4ab-2adb-4f2c-8606-d9a470834274)

I don't know much about Markdown but can we have a template for RTL languages with direction=rtl?
if not we have another a way that is use Farsi equivalent of word instead of English, for example:
IMG = ایمیج
.txt = تی اکس تی.
and because this isn't very common in our region this may cause confusion.

but for arguments there is two ways ( to my knowledge ):
1- write arguments in reverse order to print correctly ( too genious :-) )
2- use english arguments

Reply:
Yeah, I noticed the issues with file extensions too, currently done converting Farsi placeholders to English placeholders locally, starting for Arabic pages (when I open a PR will ping you for a review).

Reply:
Sure! thanks for your help.